source/production/F0.Minesweeper.Components/Minefield.razor

```diff
@@ -0,0 +1,23 @@
+<h3>Minefield</h3>
+
+@if (isValidSize)
+{
+	<table>
+		@for (uint currentHeight = 0; currentHeight < @Size.Height; currentHeight++)
+		{
+			<tr>
+				@for (uint currentWidth = 0; currentWidth < @Size.Width; currentWidth++)
+				{
+					var location = new F0.Minesweeper.Components.Abstractions.Location(currentWidth, currentHeight);
+					<td>
+						<Cell Location="location" />
+					</td>
+				}
+			</tr>
+		}
+	</table>
+}
+else
+{
+	<label>Minesweeper is played on a Minefield and not within a black hole! Provide a valid size!</label>
```

Comment:
Maybe we could leave this in as a kind of easter egg when we later on provide a feature where the users can enter the minefield size on their own?